packages/forms/src/fields/other/FauxInput.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable react/prop-types */
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { composeEventHandlers } from '@zendeskgarden/container-selection';
+import { StyledTextFauxInput } from '../../styled';
+
+const VALIDATION = {
```

Comment:
Should the repeated VALIDATION const be moved out to `utils`?

Reply:
I've abstracted out the `VALIDATION` const to a utility for the styled elements. For the exported components (Input, Textarea, Message, etc.) the const still needs to live within the component to allow styleguidist to display the strings correctly.